.jest-bazel.config.js

```diff
@@ -0,0 +1,22 @@
+// @flow strict
+
+global.__DEV__ = true;
+
+// TODO: should we rather have one per each project to isolate them properly?
```

Comment:
We can use jest-presets maybe?

Reply:
How do you mean it? I was thinking about removing this global config and basically copy-pasting it everywhere to have truly independent repositories. This file creates a global dependency for all the repositories, which seems to be a bad practice in the monorepos world.

Reply:
I mean, you can have a jest preset at the root, and extend it or overwrite in each workspace. 

Surely, there will be some commons, right?

https://jestjs.io/docs/en/configuration#preset-string

Reply:
Ah, I understand now. Sure, can be. :)